tooling/bundler/src/bundle/windows/util.rs

```diff
@@ -27,7 +27,16 @@ pub const WIX_UPDATER_OUTPUT_FOLDER_NAME: &str = "msi-updater";
 
 pub fn download(url: &str) -> crate::Result<Vec<u8>> {
   info!(action = "Downloading"; "{}", url);
-  let response = ureq::get(url).call().map_err(Box::new)?;
+
+  let mut agent_builder = ureq::AgentBuilder::new();
+  if let Ok(proxy_url) = std::env::var("http_proxy") {
```

Comment:
instead of reading the env var, you should be using `AgentBuilder::try_proxy_from_env` https://docs.rs/ureq/latest/ureq/struct.AgentBuilder.html#method.try_proxy_from_env